core/components/atoms/_chip/index.tsx

```diff
@@ -58,7 +58,7 @@ export const GenericChip = (props: GenericChipProps) => {
   const renderLabel = () => {
     if (typeof label === 'string') {
       return (
-        <Text data-test="DesignSystem-GenericChip--Text" color={textColor}>
+        <Text data-test="DesignSystem-GenericChip--Text" color={textColor} className="Chip-text">
```

Comment:
Please add a test case to check `Chip-text` class is applied only when label is passed as `string`